### PR TITLE
synthetics features for R262

### PR DESCRIPTION
**Pull request by andreachiavegatto**
- Documenting usage of synthetics credentials 
- Documenting filter property `synthetic.testActive`

[Story/Card](https://instana.kanbanize.com/ctrl_board/132/cards/143654/details/)

### CODE REVIEW

File: spec/descriptions/tagSyntheticResults.md [hunk 1]
```diff
@@ -166,6 +166,15 @@ Either tagFilters or tagFilterExpression can specify a custom property by its ke
...
 
+To filter tests by their active state (`true`/`false`) use the custom property label `synthetic.testActive`.
+```
+"tagFilters": [{ 
+  "name":"synthetic.testActive", 
+  "operator":"EQUALS",
+  "booleanValue": false 
+}]
+```
+
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticResults.md:177]: synthetic.testActive is only applicable to getTestSummaryList API call, so this change and the change below need to go to the section: Get a list of Synthetic tests with Success Rate and Average Response Time data.

File: spec/descriptions/tagSyntheticResults.md [hunk 2]
```diff
@@ -205,6 +214,10 @@ Either tagFilters or tagFilterExpression can specify a custom property by its ke
       "name": "synthetic.locationId", 
       "operator": "EQUALS", 
       "stringValue": "abcdefgXSJmQsehOWg1S"
+    }, {
+      "name": "synthetic.testActive",
+      "operator": "EQUALS",
+      "booleanValue": true
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticResults.md:220]: synthetic.testActive is only applicable to getTestSummaryList API call, so this change and the change above need to go to the section: Get a list of Synthetic tests with Success Rate and Average Response Time data.

File: spec/descriptions/tagSyntheticSettings.md [hunk 3]
```diff
@@ -98,3 +98,19 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
...
 - **testFrequency** How often the playback for a Synthetic test is scheduled. The unit of the testFrequency parameter is minute.
   The default is every 15 minutes. The range is from 1 minute to 120 minutes.
+
+## Synthetic Credentials:
+
+Synthetics credentials can be used to store passwords and/or secrets used by the Synthetic Tests.
+
+All Script Tests can use credentials on their body and API Simple Tests can use them on header parameters.
+
+It is required that the credentials used in the test be created before the test is created or modified to use them.
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticSettings.md:108]:
```suggestion
It is required that the credentials used in the test be created before the test is created or modified.
```

File: spec/descriptions/tagSyntheticSettings.md [hunk 4]
```diff
@@ -98,3 +98,19 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
...
+
+Tests using credentials are validated during test creation and update whether you use the API or UI to create the test, as foillowing:
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticSettings.md:110]:
```suggestion
Tests using credentials are validated during test creation and update whether you use the API or UI, as foillowing:
```

File: spec/descriptions/tagSyntheticSettings.md [hunk 5]
```diff
@@ -98,3 +98,19 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
...
+
+1. The user Id of the logged in user or API Token being used to create or modify the test must have permission to use credentials.  
+  API requests to create or update a test referencing credentials without the correct permission will fail with return code `Forbidden`.
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticSettings.md:113]:
```suggestion
  Requests to create or update a test referencing credentials without the correct permission will fail with return code `Forbidden`.
```

File: spec/descriptions/tagSyntheticSettings.md [hunk 6]
```diff
@@ -98,3 +98,19 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
...
+
+2. The credentials or secrets used in the test must exist.  
+  API requests to create or update a test referencing credentials that do not exist will fail with return code `Bad Request`.
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticSettings.md:116]:
```suggestion
  Requests to create or update a test referencing credentials that do not exist will fail with return code `Bad Request`.
```

File: spec/descriptions/tagSyntheticSettings.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-The API endpoints of this group can be used to manage Synthetic Locations and Synthetic Tests. 
+The API endpoints of this group can be used to manage Synthetic Locations, Synthetic Tests and Synthetics Credentials.
```
Comment (KarenGdaniec) [inline spec/descriptions/tagSyntheticSettings.md:1]:
```suggestion
The API endpoints of this group can be used to manage Synthetic Locations, Synthetic Tests and Synthetic Credentials. 
```

File: spec/descriptions/tagSyntheticSettings.md [hunk 8]
```diff
@@ -98,3 +98,19 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
   Synthetic tests run at all locations simultaneously. 
 - **testFrequency** How often the playback for a Synthetic test is scheduled. The unit of the testFrequency parameter is minute.
   The default is every 15 minutes. The range is from 1 minute to 120 minutes.
+
+## Synthetic Credentials:
+
+Synthetics credentials can be used to store passwords and/or secrets used by the Synthetic Tests.
```
Comment (KarenGdaniec) [inline spec/descriptions/tagSyntheticSettings.md:104]:
```suggestion
Synthetic credentials can be used to store passwords and/or secrets used by the Synthetic Tests.
```

File: spec/descriptions/tagSyntheticSettings.md [hunk 9]
```diff
@@ -98,3 +98,19 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
...
+
+All Script Tests can use credentials on their body and API Simple Tests can use them on header parameters.
```
Comment (KarenGdaniec) [inline spec/descriptions/tagSyntheticSettings.md:106]:
```suggestion
All Script Tests can use credentials in their body and API Simple Tests can use them on header parameters.
```

File: spec/descriptions/tagSyntheticSettings.md (hunk 6 above)
Comment (KarenGdaniec) [reply spec/descriptions/tagSyntheticSettings.md:116]: Good suggestion, Carol.  I will modify the release notes as well.

**Review by KarenGdaniec — CHANGES_REQUESTED** [review]
see comments

**Review by carolzhang65 — APPROVED** [review]
The changes look good to me. Thanks.

File: spec/descriptions/tagSyntheticSettings.md [hunk 10]
```diff
@@ -98,3 +98,19 @@ The API endpoints of this group can be used to manage Synthetic Locations and Sy
...
+
+## Synthetic Credentials:
+
+Synthetic credentials can be used to store passwords and/or secrets used by the Synthetic Tests.
+
+All Script Tests can use credentials in their body and API Simple Tests can use them on header parameters.
+
+It is required that the credentials used in the test be created before the test is created or modified.
+
+Tests using credentials are validated during test creation and update whether you use the API or UI, as following:
```
Comment (KarenGdaniec) [inline spec/descriptions/tagSyntheticSettings.md:110]:
```suggestion
Tests using credentials are validated during test creation and update whether you use the API or UI, as follows:
```

**Review by KarenGdaniec — APPROVED** [review]
Looks good

**Pull request merged**